docs/design/classes.md

```diff
@@ -701,10 +706,23 @@ class TextLabel {
 }
 ```
 
-The main difference here is that `"default"` is a default instead of an
-initializer, and will be ignored if another value is supplied for that field
-when constructing a value. Defaults must be constants whose value can be
-determined at compile time.
+Notice that this is subtly different from the meaning of `var` in other
+contexts: it declares an
+[instance variable](https://en.wikipedia.org/wiki/Instance_variable)), not just
+a variable in the class's scope.
+
+> **Open question:** Is there a way to declare a variable in a class's scope?
+
+In a field declaration, an initializer (such as `= "default` above) specifies
+the default value of the field, and will be ignored if another value is supplied
+for that field when constructing an instance of the class. Defaults must be
+constants whose value can be determined at compile time.
+
+The pattern in a field declaration must be a run-time binding pattern, so the
+full syntax is:
+
+_field-declaration_ ::= `var` _identifier_ `:` _expression_ [ `=` _expression_
+] `;`
```

Comment:
I think it would be nice to clarify the different expressions here: the first is implicitly cast to `type`, and the second needs to have the initializing category.
```suggestion
_field-declaration_ ::= `var` _identifier_ `:` _type-expression_ [ `=`
    _init-expression_ ] `;`
```

Reply:
This is intended to be a grammar rule, so the names in italics are nonterminal symbols of the grammar. I don't think it makes sense to have separate nonterminals for type expressions and init expressions, since as far as I can tell, they'd have identical definitions.